src/components/about-us/section-01/index.js

```diff
@@ -0,0 +1,147 @@
+import React, { PureComponent } from 'react'
+import styled from 'styled-components'
+import { screen } from '../utils/screen'
+import { content } from '../constants/data/section1Content'
+import Title from './title'
+
+const containerWidth = {
+  mobile: '100%',
+  tablet: '719px',
+  desktop: '1024px',
+  overDesktop: '1440px'
+}
+
+const ContainerWrapper = styled.div`
+  position: relative;
+  margin: 0 auto;
+  background-color: white;
+  overflow: hidden;  
+  ${screen.desktopAbove`
+    width: ${containerWidth.desktop};
+  `}
+  ${screen.overDesktop`
+    width: ${containerWidth.overDesktop};
+  `}
+  ${screen.tablet`
+    width: ${containerWidth.tablet}
+  `}  
+`
+
+const SectionBlock = styled.div`
+  display: flex;
+  justify-content: space-between;
+  width: 100%;
+  ${screen.mobile`
+    flex-direction: column;
+    align-items: center;
+  `}
+`
+
+const Oneblock = styled.div`
```

Comment:
what is `Oneblock` ?
Can not tell what it means by name.

Reply:
It means a feature which is displayed in a block ( a container ) . I will change the name to FeatureBlock, maybe it would be more easy to understand.